include/operations/blas2_trees.h

```diff
@@ -501,15 +501,15 @@ GerCol<Single, Lower, Diag, Upper, lhs_t, rhs_1_t, rhs_2_t> make_ger_col(
       lhs_, scalar_, rhs_1_, rhs_2_, nWG_row_, nWG_col_, local_memory_size_);
 }
 
-/**** GERP N COLS x (N + 1)/2 ROWS FOR PACKED MATRIX ****/
+/**** SPR N COLS x (N + 1)/2 ROWS FOR PACKED MATRIX ****/
```

Comment:
I think we can change this documentation to something like this:
`/**** SPR N COLS x (N + 1)/2 ROWS FOR PACKED MATRIX ****/` 
`/*This class performs rank 1/2 update for symmetric packed matrices. For more details on matrix ...`

The existing documentation won't make much sense now that we have changed the class name.

Reply:
Absolutely, I update it.